internal/provider/resource_aci_route_control_profile_test.go

```diff
@@ -103,7 +99,6 @@ func TestAccResourceRtctrlProfileWithFvTenant(t *testing.T) {
 					resource.TestCheckResourceAttr("aci_route_control_profile.test", "owner_key", ""),
 					resource.TestCheckResourceAttr("aci_route_control_profile.test", "owner_tag", ""),
 					resource.TestCheckResourceAttr("aci_route_control_profile.test", "route_control_profile_type", "combinable"),
-					resource.TestCheckResourceAttr("aci_route_control_profile.test", "route_map_continue", "no"),
```

Comment:
Why are we removing this check?

Reply:
Introduced version_mismatch in default which takes care of this.